third_party/amd/lib/TritonAMDGPUToLLVM/Utility.h

```diff
@@ -112,6 +112,24 @@ bool isChainDotHead(mlir::triton::DotOpInterface dotOp);
 // in the same region
 bool isChainDotTail(mlir::triton::DotOpInterface dotOp);
 
+// Determine the order in which CTA tiles are laid out across the tensor.
+// That is, create vector of dimensions from fastest to slowest varying.
+SmallVector<unsigned> getCTATileOrder(MLIRContext *ctx,
```

Comment:
This function is only used in the `TritonAMDGPUDialectToLLVM` library right? We should put it in a utility file inside that library's directory.

Right now this will force `TritonAMDGPUDialectToLLVM` to depend on `TritonAMDGPUToLLVM`; and that's dependency is missing in `CMakeLists.txt` right now so compilation with parallel threads might fail actually.